### PR TITLE
WLSSB: Even even more updates

### PR DESCRIPTION
**Pull request by Lycanium-Z**
Ok this should balance everything even more than it was before

### CODE REVIEW

**Comment by Lycanium-Z** [comment]
there. everything has been tested and everything works!

**Comment by Lycanium-Z** [comment]
ok travis is fixed

File: mods/wlssb/moves.js [hunk 1]
```diff
@@ -433,40 +413,42 @@ exports.BattleMovedex = {
...
-		name: "Meteor Madness",
-		pp: 5,
-		noPPBoosts: true,
+		basePower: 60,
+		category: "Physical",
+		desc: "Has a 10% chance to inflict dread on the target.",
+		shortDesc: "10% chance to inflict dread on the target.",
+		id: "heartofdarkness",
+
+		name: "Ḫ̱̙̞̅̓͑͝e̲̅̿a̤͕̦͉ř̗̿ͫ̽͢t̼̝͊͒ͭ̀ ̖̦͆ͯỏͧf̻̙ ̺̏̂̉͐ͨ̐̃͟D̼̦͕̅ͬ̚a̝̻͉̰̯̓̊̇r͉͇̿̽ͤ̆k̍̑ͯ̃͟ͅn̺̞̤͉̪ͦ͒̒ͪ̄͑ě̴̖̙̩̘̂ŝ̸̇ͪ̃ͬŝ̨͙̙͚͎̬̂̒ͅ",
```
Comment (HoeenCoder) [inline mods/wlssb/moves.js]: No insane fonts please, I really don't want to screw something up with them.

File: mods/wlssb/moves.js [hunk 2]
```diff
@@ -433,40 +413,42 @@ exports.BattleMovedex = {
...
 		},
-		onHit: function (target, source) {
-			this.heal(Math.ceil(target.maxhp * 0.5), source);
-			let oldAbility = target.setAbility('normalize');
-			if (oldAbility) {
-				this.add('-ability', target, 'Normalize', '[from] move: Meteor Madness');
-				return;
-			}
+		onHit: function (target) {
+			this.add('c|%Lycanium Z|T̡̞͚͈̥̋ͧͯ̈̓ͩ̆i̗̱̦ͦͦͅm̞̥̰͎̱ͬ̅̆̈̽e̢͕̞̙͆ͤͫ ̲̳̮ͩ̐͌̈̔͐͗͢t̗ͤ̂̃o̱̖̤̤̟͔͛̐ͩ̌ ͍͚̺͍̹̓ͨs̢̞̮̏ḫ͖̱̖̍́ͮͧ̚ò̷͔ͫw̨̞͕̻̓̒͋ͮ ̧ͣ̚y͈͚͉̥̳̒́̇͌̓̈̑ŏ̼͇̠̰̖̈́̅ͧ̓͗́u̴͇̟̔̆͆͂ ͔̖̼͔̣̗̲͋̆̊̈̊̓̊͢w̹̻͔̠̙̫ͫͧ͛̅̑h͚̣̳̮̪͌̉ͥ̀̀a̽͏̘̘͕̹t̤̟̝̘̺͉̼̽ ̥̻̳͖̮̅ͤ̏͌͗ͅi̼͍̍́͢t͉̙̳̹ͤ͝s̺͒ͨ̔̋̓͛ ̺͈̹̻̟̱̞̅ͩͦ͋̋̾͛lͯ҉̪͉̝̻̙i̗̺͖̞ͅk͗̎͑̅ͯ҉̭e̅̊ͥ̄̑ͭͩ ̸̩̜̼̀ͫ̿͗̃ͅt̛̠̑̍ͤͤͩͬ̚ǒ̲̜̼͕̞ ̋̾͛̃̎̾͡d̟̯̤͚̤̬̐͟įĕ̻̞̖͖͎̿͛!̞̳̘̳͒̇̋̈');
```
Comment (HoeenCoder) [inline mods/wlssb/moves.js]: re: weird fonts

**Pull request merged**